Error when Starting Docker Image

Hello All 
Just starting seeing this message time I start and stop the image
xteve_lazystream_1  | 2021/03/30 15:35:08 [xTeVe] Version:               2.1.2 Build: 0121
xteve_lazystream_1  | 2021/03/30 15:35:08 [xTeVe] Database Version:      2.1.0
xteve_lazystream_1  | 2021/03/30 15:35:08 [xTeVe] GitHub:                https://github.com/xteve-project
xteve_lazystream_1  | 2021/03/30 15:35:08 [xTeVe] Git Branch:            master [xteve-project]
xteve_lazystream_1  | 2021/03/30 15:35:08 [xTeVe] SSDP / DLNA:           true
xteve_lazystream_1  | 2021/03/30 15:35:09 [xTeVe] Update Available:      Version: 2.2.0.0200
xteve_lazystream_1  | 2021/03/30 15:35:09 [xTeVe] Update Server:         GitHub
xteve_lazystream_1  | 2021/03/30 15:35:09 [xTeVe] Start Update:          Branch: master
xteve_lazystream_1  | 2021/03/30 15:35:09 [ZIP] New version (xteve_2): 2.2.0.0200
xteve_lazystream_1  | 2021/03/30 15:35:10 [ZIP] Download new version...
xteve_lazystream_1  | 2021/03/30 15:35:10 [xTeVe] [ERROR] open /usr/bin/xteve: permission denied (Update failed) - EC: 6002

I'm also see the 2021-04-02 15:50:54 [xTeVe] [ERROR] open /usr/bin/xteve: permission denied (Update failed) - EC: 6002 and plex no longer appears to be updating the DVR guides

> I'm also see the 2021-04-02 15:50:54 [xTeVe] [ERROR] open /usr/bin/xteve: permission denied (Update failed) - EC: 6002 and plex no longer appears to be updating the DVR guides

Mine still is working it is just when I restart my docker that mine happens. Normally when It is not updating the guide you have to add the connection back into plex and that kind of kick starts it.

--- a/root/etc/cont-init.d/60-chown-files
+++ b/root/etc/cont-init.d/60-chown-files
@@ -7,3 +7,6 @@ chown -R abc:abc /xteve
 chown -R abc:abc /tmp/xteve
 
 chown -R abc:abc /home/abc
+
+# So xteve can auto update
+chown -R abc:abc /usr/bin

